.circleci/config.yml

```
@@ -263,6 +263,8 @@ jobs:
           name: Py-compile packages
           command: |
             pyodide py-compile --compression-level 0 dist/
+            rm dist/snapshot.bin
```

nit:

```suggestion
            rm -f dist/snapshot.bin
```

For safety. (don't raise error even if snapshot.bin does not exist).